packages/builder/package.json

```diff
@@ -16,6 +16,7 @@
     "fs-extra": "^7.0.1",
     "glob": "^7.1.3",
     "hash-sum": "^1.0.2",
+    "ignore": "^5.0.4",
```

Comment:
Must be in `@nuxt/utils`

Reply:
As @pi0 mentioned, maybe we should move ignore back to build for bundle size.